__tests__/app.test.js

```diff
@@ -0,0 +1,45 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const categoryData = require("../db/data/development-data/categories");
+const commentData = require("../db/data/development-data/comments");
+const reviewData = require("../db/data/development-data/reviews");
+const userData = require("../db/data/development-data/users");
+const seed = require("../db/seeds/seed");
+
+afterAll(() => {
+  db.end();
+});
+
+beforeEach(() => {
+  return seed({ categoryData, commentData, reviewData, userData });
+});
+
+describe("app", () => {
+  describe("/api", () => {
+    it("200: GET - responds with server OK message", () => {
+      return request(app) //ARRANGE
+        .get("/api") // ACT
+        .expect(200) //ASSERT ---- Supertest
+        .then((response) => {
+          expect(response.body.msg).toBe("all ok");
+        });
+    });
+  });
```

Comment:
This test would be redundant as your next test also expects 200, and will only pass with that status code